openstack/objectstorage/v1/containers/testing/fixtures.go

```diff
@@ -82,38 +82,14 @@ func HandleListContainerInfoSuccessfully(t *testing.T) {
 	})
 }
 
-// HandleListContainerNamesSuccessfully creates an HTTP handler at `/` on the test handler mux that
-// responds with a `ListNames` response when only container names are requested.
-func HandleListContainerNamesSuccessfully(t *testing.T) {
```

Comment:
This fixture is not used any more, because we don't send `accept: text/plain` any more on listing.